doc/man3/SSL_CTX_set_mode.pod

```diff
@@ -105,6 +105,16 @@ Enable asynchronous processing. TLS I/O operations may indicate a retry with
 SSL_ERROR_WANT_ASYNC with this mode set if an asynchronous capable engine is
 used to perform cryptographic operations. See L<SSL_get_error(3)>.
 
+=item SSL_MODE_NO_KTLS_TX
+
+Disable the use of the kernel TLS egress data-path. By default kernel TLS is
```

Comment:
Since many users can't use this. It is disabled by default on compilation (see Configure). If however, a user has compiled OpenSSL with enable-ktls, then I assume that it should be ON by default. This API allows fine-grained control on whether to use KTLS on a specific socket.

Reply:
Ah, right. Ok that makes sense. In which case we should be careful about the wording here then. Its only on by default if ktls support has been enabled. We should also document the restrictions that apply if KTLS is being used, i.e. no renegotiation, no setting the max fragment size, etc.

Reply:
The restrictions are not inherent, but rather temporary until they are implemented. I'll add a comment about the current status, but this may change over time.